src/scripts/main.js

```diff
@@ -1,3 +1,64 @@
 'use strict';
 
 // write code here
+const table = document.querySelector('table');
+const tableBody = table.tBodies[0];
+const employees = getEmployees(tableBody.rows);
+
+document.addEventListener('click', (e) => {
+  const tableHeading = e.target.closest('th');
+
+  if (!tableHeading || tableHeading.closest('tFoot')) {
+    return;
+  }
+
+  const sortKey = tableHeading.textContent.trim('').toLowerCase();
+  const compareFunction = createSorterByKey(sortKey);
```

Comment:
too long a way to scroll to get the logic of this compareFunction. You may separate a helper functions to a different files btw